package.json

```diff
@@ -28,7 +28,7 @@
   ],
   "scripts": {
     "presystem-test": "npm run compile",
-    "system-test": "mocha build/system-test --timeout 600000",
+    "system-test": "GRPC_TRACE=all GRPC_VERBOSITY=DEBUG npx mocha build/system-test --timeout 600000",
```

Comment:
the `npx` will cause the latest version of mocha from npm to always be used, which I"m slightly concerned with from a supply chain point of view.

What I might perhaps do instead is use [cross-env](https://www.npmjs.com/package/cross-env), so that system-tests keep working on Windows, and drop the npx:

```
cross-env GRPC_TRACE=all GRPC_VERBOSITY=DEBUG mocha build/system-test --timeout=6000000
```

Reply:
Yeah, sorry, I forgot to mention in chat that these particular diffs are just debug stuff I'll be removing. I'm trying to see if the DEADLINE_EXCEEDED error can be reproduced here.